### PR TITLE
AL-19: Test coverage

### PR DESCRIPTION
**Pull request by iterative-it**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/event/DomainEventPublisherTest.kt [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import java.util.UUID
+
+class DomainEventPublisherTest {
+  private val objectMapper = testObjectMapper()
+
+  private val baseUrl = "http://localhost:8080"
+
+  @Test
+  fun `publish alert event`() {
+    val hmppsQueueService = mock<HmppsQueueService>()
```
Comment (jbowdidge-moj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/event/DomainEventPublisherTest.kt:29]: Should these mocks be fields on the class?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/event/AlertEventServiceTest.kt [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import java.time.LocalDateTime
+import java.util.UUID
+
+class AlertEventServiceTest {
+  private val baseUrl = "http://localhost:8080"
+
+  @Test
+  fun `handle alert event - publish enabled`() {
+    val eventProperties = EventProperties(true, baseUrl)
+    val telemetryClient = mock<TelemetryClient>()
```
Comment (jbowdidge-moj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/event/AlertEventServiceTest.kt:29]: Same here?

**Review by jbowdidge-moj — COMMENTED** [review]
Just one or two small comments...

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/event/DomainEventPublisherTest.kt (hunk 1 above)
Comment (iterative-it) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/event/DomainEventPublisherTest.kt]: Good shout. Moved

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/event/AlertEventServiceTest.kt (hunk 2 above)
Comment (iterative-it) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/event/AlertEventServiceTest.kt]: As above

**Review by jbowdidge-moj — APPROVED** [review]
LGTM

**Pull request merged**